server/admin_management/api/app/main.py

```diff
@@ -82,7 +78,12 @@ def custom_generate_unique_id(route: APIRouter):
 app.include_router(
     router_access_control_privilege.router,
     prefix=apiPrefix + "/access_control_privileges",
-    tags=["FAM Access Control Privilege"],
+    tags=["FAM Access Control Privileges"],
+)
+app.include_router(
+    router_fam_admin_user_access.router,
+    prefix=apiPrefix + "/fam-admin-user-accesses",
+    tags=["FAM Admin User Accesses"],
```

Comment:
As per comment in conversation, likely /fam-access-manager-accesses is clearer.

Reply:
Maybe we can include this change and do it at the ticket @MCatherine1994  created for endpoints renaming?
I am not even sure "/fam-**access-manager**-accesses" is clear to everyone either. For example, it is already confusing on confluence these business naming and database table naming, because other than regular users access, everything else seem to make sense to be FAM's **admin**(s) at different levels; and maybe "-access-manager-" some what make people confuse with "Admin Manager" on confluence's term and table naming.
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/b367a97a-544c-485f-9ef6-bd5602896c1d)

Reply:
Or maybe I can adjust it from "**fam**-admin-[...]" to be "**/admin-user-accesses**" without "**fam**" in the path.
I feel that is better not to make confusion to the term FAM_ADMIN or FAM Admin.

Reply:
Yes, dropping the fam- would also fix the problem.

Reply:
Thanks Basil!! Naming has been refactored and update the endpoint path on confluence as well.